packages/perseus/src/server-item-renderer.tsx

```diff
@@ -51,11 +52,9 @@ type HOCProps = {
 
 type Props = OwnProps & HOCProps;
 
-type DefaultProps = {
-    apiOptions: Props["apiOptions"];
-    onRendered: Props["onRendered"];
-    linterContext: Props["linterContext"];
-};
+type DefaultProps = Required<
+    Pick<Props, "apiOptions" | "onRendered" | "linterContext">
```

Comment:
Not asking for changes, but I actually find the original easier to read as opposed to the use of `Pick<>`. (I do like wrapping this all in `Required<>`)

Reply:
In this case I agree! It gets really busy when there are a lot of DefaultProps, but with just three I did feel a little silly. I was just trying to be consistent with the other files I touched.